test/data/testNMR.py

```diff
@@ -21,10 +22,36 @@ class GenericNMRTest(unittest.TestCase):
     def testsize(self):
         """Check to make sure there are the correct number of tensors parsed"""
         assert len(self.data.nmrtensors) == self.data.natom
-        assert len(self.data.nmrtensors[0]) == 3
+        assert len(self.data.nmrtensors[0]) == 4
         assert self.data.nmrtensors[0]["total"].shape == (3, 3)
+        
+        tensor = self.data.nmrtensors[0]
+        # Check the total isotropic value matches the computed value.
+        total = 0.0
+        for t_type in ("diamagnetic", "paramagnetic"):
+            total += sum(numpy.linalg.eigvals(tensor[t_type])) / len(numpy.linalg.eigvals(tensor[t_type]))
+        
+        self.assertAlmostEqual(total, tensor['isotropic'], 3)
 
 
+class GenericNMRCouplingTest(unittest.TestCase):
+    """Generic NMR spin-spin coupling unittest"""
+
+    def testsize(self):
+        """Check to make sure there are the correct number of tensors parsed"""
+        assert len(self.data.nmrcouplingtensors) == 139
+        
+        tensor = list(list(self.data.nmrcouplingtensors.values())[0].values())[0]
+        assert len(tensor) == 7
+        assert tensor['total'].shape == (3, 3)
+        
+        # Check the total isotropic value matches the computed value.
+        total = 0.0
+        for t_type in ("diamagnetic", "paramagnetic", "fermi", "spin-dipolar", "spin-dipolar-fermi"):
+            total += sum(numpy.linalg.eigvals(tensor[t_type])) / len(numpy.linalg.eigvals(tensor[t_type]))
+        
+        self.assertAlmostEqual(total, tensor['isotropic'], 3)
```

Comment:
We are migrating away from unittest to pure pytest and will no longer be able to use unittest methods.  Pure pytest mostly uses bare asserts, though it does come with some numeric helpers like [`approx`](https://docs.pytest.org/en/stable/reference/reference.html#pytest-approx).  The initial translation for `assertAlmostEqual` looks like https://github.com/cclib/cclib/pull/1186/files#diff-691e094e560c34e2a66142c5f70a2b6a8a733a69ad4db43c4eaf106b96f303cbR84 because I used an automated tool to do it.  Can you switch these to `pytest.approx`?

Reply:
Can do